exotations/task_maps/task_map/init/JointAccBackFiniteDiff.in

```diff
@@ -0,0 +1,4 @@
+extend <exotica/TaskMap>
+
+Required Eigen::VectorXd JointState_prev_1;
```

Comment:
Can be made ``Optional`` and default state assigned, i.e.: ``Optional Eigen::VectorXd JointState_prev_1 = Eigen::VectorXd();``. Then in the Initialize method check whether size is 0, then assign zero-vector-of-length-N, else check if size equals N, then assign, else throw. This comment applies to all initializers.

Reply:
I have removed these completely from `.in` files and added a `initPrevJointState` function that must be called to init each task map.